mirage/views/user.ts

```diff
@@ -30,6 +30,23 @@ export function userRegistrationList(this: HandlerContext, schema: Schema, reque
     return json;
 }
 
+export function userPreprintList(this: HandlerContext, schema: Schema, request: Request) {
+    const user = schema.users.find(request.params.id);
+    const preprints = [];
+    const { contributorIds } = user;
+
+    for (const contributorId of contributorIds as string[]) {
+        const contributor = schema.contributors.find(contributorId);
+        const preprint = contributor.preprint;
+        if (preprint && filter(preprint, request)) {
+            preprints.push(this.serialize(preprint).data);
+        }
+    }
+
+    const json = process(schema, request, this, preprints, { defaultSortKey: 'dateModified' });
```

Comment:
I noticed the other views use last_logged as the defaultSortKey so I have a couple questions. Should this be last_logged as well? If not, should it be snake_case?

Reply:
Updated that.